Automattic/wp-calypso
# Updates the colours used in the Store on WP.com to use the latest brand colours

#### Changes proposed in this Pull Request

* This PR updates the colours used for the illustrations in Store on WordPress.com to be in line with the colours used in Colour Studio.

#### Testing instructions

- Some of these illustrations are during the setup of a new Store on WordPress.com, so you would need to create a new site, with a Business plan, install any plugin and then choose to start setting up a Store via the menu item in the sidebar - these are the two to look for:

<img width="508" alt="Screenshot 2019-06-12 at 14 43 26" src="https://user-images.githubusercontent.com/10805911/59388395-78670c80-8d20-11e9-98ad-e53df3350d6d.png">

Once you have set up a new store - and visit the dashboard you will see this illustration:

<img width="792" alt="Screenshot 2019-06-12 at 14 44 40" src="https://user-images.githubusercontent.com/10805911/59388460-a2203380-8d20-11e9-8e94-efab9606cef7.png">

I could not locate the use of these other illustrations in Store on WordPress.com - but I updated them none the less:

<img width="613" alt="Screenshot 2019-06-12 at 14 47 04" src="https://user-images.githubusercontent.com/10805911/59388575-f75c4500-8d20-11e9-9bc4-69623e22e8bf.png">


Fixes: #33259
